### PR TITLE
Remove fastclick: it's obsolete and causing issues with <select>

### PR DESCRIPTION
**Pull request by diox**
On my Android device, this fixes issues with `<select>` and doesn't regress tapping on links. Please test on yours.

Fixes https://github.com/mozilla/addons-frontend/issues/11135

### CODE REVIEW

**Comment by eviljeff** [comment]
> On my Android device, this fixes issues with `<select>` and doesn't regress tapping on links. Please test on yours.

Are there instructions to do that?

**Comment by diox** [comment]
https://github.com/mozilla/addons-frontend#configuring-an-android-device-for-local-development

File: src/amo/client/base.js [hunk 1]
```diff
@@ -15,7 +14,6 @@ import tracking from 'amo/tracking';
 export default async function createClient(
   createStore,
   {
-    _FastClick = FastClick,
```
Comment (bobsilverberg) [inline src/amo/client/base.js:18]: This is referenced in the test setup for https://github.com/mozilla/addons-frontend/blob/master/tests/unit/amo/client/test_base.js, so we should probably remove any code related to `FastClick` from that test file.

**Review by bobsilverberg — APPROVED** [review]
I don't have any Android device to test this on, but the code looks fine to me, other than my comment about the test file.

r+wc

**Comment by diox** [comment]
@bobsilverberg can you test on iPhone ? Fastclick has its own logic with user agent detection and other checks, so it may have been applied to other browsers/devices. So it'd be good to see if you can notice a difference without it locally on whatever device you have access to.

**Comment by diox** [comment]
Do you see any delay clicking on links before/after those changes ?

**Pull request merged**